docs/learn/learn-xcm-pallet.md

```diff
@@ -8,14 +8,119 @@ slug: ../learn-xcm-pallet
+   to check the message and ensure that no barrier/filter will block the execution of the XCM. Once
+   it is deemed valid, the message will then be _locally_ executed, therein returning the outcome as
+   an event. This operation is executed on behalf of whichever account has signed the extrinsic.
+   It's possible for only a partial execution to occur.
+2. `send` - This call specifies where a message should be sent externally to a particular
+   destination, i.e. a parachain, smart contract, or any system which is governed by consensus. As
+   before with `execute`, the Executor checks the origin, destination, and message and is then sent
+   to the `XcmRouter`.
+
+:::info
+
+The XCM pallet needs the `XcmRouter` to send XCMs. It is used to dictate where XCMs are allowed to
+be sent, and which XCM transport protocol to use. For example, Kusama, the canary network, uses the
+`ChildParachainRouter` which only allows for Downward Message Passing from the relay to parachains
+to occur.
+
+You can read more about [XCM transport methods here](./learn-xcm-transport.md).
+
+:::
+
+## Asset Transfer Extrinsics
+
+Several extrinsics within the pallet handle asset transfer logic. They define a predetermined set of
+instructions for sending and executing XCMs. Two variants of these functions are prefixed with
+`limited_`. They have the same functionality but can specify a weight to pay for the XCM fee.
+
+Otherwise, the fee is taken as needed from the asset being transferred.
+
+1. `reserve_transfer_assets` - Transfer some assets from the local chain to the sovereign account of
+   a destination chain and forward an XCM containing a `ReserveAssetDeposited` instruction, which
+   serves as a notification.
+2. `teleport_assets` - Teleport some assets from the local chain to some destination chain.
+
+### Transfer Reserve vs. Teleport
+
+While both extrinsics deal with transferring assets, they exhibit fundamentally different behavior.
+
+- **Teleporting** an asset implies a two-step process: the asset is burned/destroyed in the origin
+  chain and re-minted to whatever account is specified at the destination. Teleporting should only
+  occur if there is an inherent and bilateral trust between the two chains, as the tokens destroyed
+  _could not_ necessarily be guaranteed to have the same properties when minted at the destination,
+  as well as that there has to be **trust** that the a particular chain burned, or re-minted the
+  assets.
+- **Transferring** or **reserving** an asset implies that **equivalent** assets (i.e, native
+  currency, like `DOT` or `KSM`) are withdrawn from _sovereign account_ of the origin chain and
+  deposited into the sovereign account on the destination chain. Unlike teleporting an asset, it is
+  not destroyed and re-minted, rather a trusted, 3rd entity is used (i.e., Statemint on Polkadot) to
+  **reserve** the assets, wherein the sovereign account on the destination chain obtains ownership
```

Comment:
The sovereign account is located on the reserve chain, not the destination chain.

Reply:
I think you meant to say:
```suggestion
  **reserve** the assets, wherein the sovereign account of the destination chain on the reserve chain obtains ownership
```